File: 백준/Silver/1312. 소수/소수.py
import sys

sys.setrecursionlimit(10**6)
input = sys.stdin.readline


def main():
    a, b, n = map(int, input().split())

    for _ in range(n):
        a = (a % b) * 10
        res = a // b

    print(res)


if __name__ == "__main__":
    main()

File: 백준/Silver/1312. 소수/소수.c
#include <stdio.h>

int main()
{
    int a, b, n, res;
    scanf("%d %d %d", &a, &b, &n);
    for (int i = 0; i < n; i++)
    {
        a = (a % b) * 10;
        res = a / b;
    }
    printf("%d", res);
}